Samples Update

abapsheep opened the pull request:
z2ui5_cl_demo_app_067 - added numeric Conversion

z2ui5_cl_demo_app_326 - was not added to samples_000 this sample will dump! /SCWM/TANUM is a NUMC12 with an conversion exit ALPH0. NUMC12 in an external format (000000000099 -> 99) can not be serialized.

Commit:
Samples Update 

z2ui5_cl_demo_app_067 - added numeric Conversion

z2ui5_cl_demo_app_326 - was not added to samples_000 this sample will dump! /SCWM/TANUM is a NUMC12 with an conversion exit ALPH0. NUMC12 in an external format (000000000099 -> 99) can not be serialized.

Commit:
update

oblomov-dev commented on src/z2ui5_cl_demo_app_067.clas.abap:
maybe better use type n length 10?

oblomov-dev commented on src/z2ui5_cl_demo_app_326.clas.abap:
```diff
@@ -0,0 +1,121 @@
+    INTERFACES z2ui5_if_app.
+
+    DATA unit              TYPE meins.
+    DATA numc              TYPE /SCWM/TANUM.
```
to reduce dependancies we better make an dictoionary type out of it?

oblomov-dev committed:
Merge branch 'standard' into main

oblomov-dev commented:
updated the branch with latest changes from https://github.com/abap2UI5/samples/pull/493

Don't miss to pull these changes before continuing...

Commit:
Changed Dataelement

Created Dataelement Z2UI5_NUMC12 to showcase conversion of Numerics.

Z2UI5_NUMC12:

Data Type        NUMC
No. Characters       12
Decimal Places        0
Routine          ALPH0

Commit:
Dataelement Z2UI5_NUMC12

Commit:
Update

oblomov-dev committed:
Update ABAP_STANDARD.yaml

oblomov-dev commented:
thanks
